### PR TITLE
feat(scrypto): hello unix blueprint

### LINKED ISSUE #49: Task: Unix scrypto blueprint
**Describe the new feature**
Define Unix Blueprint in Scrypto using the Hello blueprint as a reference.

**Where the feature should be placed**
Under `scrypto` similar as demonstrated with hello

**Expected behaviour**
Just an empty blueprint with out any logic yet.

**Additional context**
Be sure to run with nx `pnpm run ci`

### PR DESCRIPTION
**Pull request by attilacsanyi**
Fix #49

### CODE REVIEW

**Comment by attilacsanyi** [comment]
Just the same hello world for unix, before adding badge logic in #45

**Comment by AhsenRiaz** [comment]
Reviewed the code and everything seems great. However, the CI failed. Should I merge it? @attilacsanyi

**Comment by attilacsanyi** [comment]
Thanks @AhsenRiaz it is ok to just approve sometimes, so I know you are fine with the code.
The failure is due to long running build in CI, I tried to limit to max 15 mins to save minutes. I might need to remove the hello example so we will have only one blueprint in our repo.

**Review by AhsenRiaz — APPROVED** [review]

File: .github/workflows/unixsocial.yml [hunk 1]
```diff
@@ -98,19 +98,19 @@ jobs:
...
           # list of commands to be run on main has env flag NX_CLOUD_DISTRIBUTED_EXECUTION set to false
-          NX_CLOUD_DISTRIBUTED_EXECUTION=false npx nx-cloud record -- npx nx format:check & pids+=($!)
+          NX_CLOUD_DISTRIBUTED_EXECUTION=false pnpm nx-cloud record -- pnpm nx format:check & pids+=($!)
 
           # list of commands to be run on agents
           pnpm lint &
           pids+=($!)
 
-          pnpm test &
+          pnpm test -- --exclude=scrypto_hello,scrypto_unix &
```
Comment (attilacsanyi) [inline .github/workflows/unixsocial.yml:113]: Somehow scrypto tests never finished in CI, so I excluded for now

**Pull request merged**